src/elm/App.elm

```diff
@@ -232,7 +232,10 @@ update action model =
             then Effects.batch [ getDate, (tick 1 Tick) ]
             else Effects.none
       in
-        ( { model | tickStatus <- Waiting }
+        ( { model
+          | tickStatus <- Waiting
+          , connected <- not (model.connected)
```

Comment:
@amitaibu 
I'm toggling the `Bool` value => and then I  have a `classList` that listens to that value.
You can see the outcome on the gif image (top right led indicator is blinking).

On the current design the led light is static and I really like the original design where the led-light blinks.
It's just a design preference.